Ludicrous-Ducks/bangazon-cli
# Jk product test

# Description
product class test , product data class test

# Related Ticket(s)
#5 #22


## Testing

[y ] There are new unit tests in this PR, and I verify that there is full coverage of all new code.
[y ] I certify that all existing tests pass

## Documentation

[y ] There is new documentation in this pull request that must be reviewed..


## Steps to Test
Outline the steps to test the code in this pull request.

1. cd to Test 
2.run this command `python ProductTest.py`
3.verify all the test passed
